addons/hr_expense/static/src/scss/hr_expense.scss

```diff
@@ -19,6 +19,12 @@
     }
 }
 
+.center_attachment > .o-mail-Attachment > .o-mail-Attachment-imgContainer {
+    display: flex;
+    justify-content: center;
+    z-index: -1;
+}
```

Comment:
Instead of playing with CSS specificity with long selector, could be just `.center_attachment .o-mail-Attachment-imgContainer` (almost immediate) + use `!important`

Reply:
I'm not really a fan of using !important in css, in case someone want to override it in the future and it still works without it, just for my information why would you add an !important ? 
But for the others comment i totally agree, thanks !

Reply:
If you don't need `!important` then it's even better!

I suggested `!important` just in case you couldn't override the style when reducing the CSS specificity by removing `> .o-mail-Attachment >`

I get `!important` can be annoying when you want to override. But keep in mind here you want to override a default behaviour, and there's no reason to override on top of this override. This is a fix, and it would be wise to do some code cleaning in the future so we allow to center attachment in a easier way that with these extra LOCs in `hr_expense`.